pkg/api/logging/v1alpha2/types_test.go

```diff
@@ -0,0 +1,36 @@
+package v1alpha2
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/emicklei/go-restful"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseQueryParameter(t *testing.T) {
+	// default operation -- query
+	queryParam := "start_time=1136214245&end_time=1136214245&from=0"
+	req, err := http.NewRequest("GET", fmt.Sprintf("http://localhost/tenant.kubesphere.io/v2alpha1/logs?%s", queryParam), nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	request := restful.NewRequest(req)
+	ret, err := ParseQueryParameter(request)
+	assert.NoError(t, err)
+	assert.Equal(t, ret.Operation, OperationQuery)
```

Comment:
I think unit tests should cover all the code as much as possible, so all fields in `Parameter` need to be checked.

Reply:
As you suggested, I have added all fields in Parameter need to be checked . @wanjunlei